modules/acquia_cms_common/src/Commands/AcmsConfigImportCommands.php

```diff
@@ -567,8 +570,8 @@ public function validateConfigResetCommand(CommandData $commandData) {
     }
     // Get scope from user input.
     if ($isInteractive && empty($messages) && !$scope) {
-      $scope = $this->io()->choice(dt('Choose a scope.'), self::ALLOWED_SCOPE, NULL);
-      $commandData->input()->setOption('scope', self::ALLOWED_SCOPE[$scope]);
+      $scope = $this->io()->choice(dt('Choose a scope.'), $this->allowedScope, NULL);
```

Comment:
@chandan-singh7929 Why this changes is showing in your PR ? I think it's not from your ticket,